packages/xo-web/src/xo-app/backup/new/smart-backup.js

```diff
@@ -23,6 +26,40 @@ const VMS_STATUSES_OPTIONS = [
   { value: 'Halted', label: _('vmStateHalted') },
 ]
 
+const CustomTag = decorate([
```

Comment:
Should it be the normal behaviour of `SelectTag`? I don't know if there would be any reason not to let the user select a custom tag and if there is, we can still add a boolean prop for it.